packages/server-wallet/src/protocols/__test__/defunder.test.ts

```diff
@@ -37,6 +62,59 @@ beforeEach(async () => {
   );
 });
 
+describe('Collaborative transaction submitter', () => {
```

Comment:
I don't see a test case for:
- `txSubmitterOrder === [0, 1]`
- `participants[0]` has no stake in the outcome.

Reply:
Good point! 042749835